clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -739,10 +739,6 @@ void tools::gnutools::Linker::ConstructJob(Compilation &C, const JobAction &JA,
         else
           CmdArgs.push_back("-lsycl");
         CmdArgs.push_back("-lsycl-devicelib-host");
-        // Use of -fintelfpga implies -lOpenCL.
```

Comment:
Do we need this?
https://github.com/intel/llvm/blob/fb70c93770981a9ea1ae74d8541de56a150078c9/clang/lib/Driver/ToolChains/Gnu.cpp#L311

Reply:
Oh, good catch - this needs to be removed.